events/src/gateway.rs

```diff
@@ -0,0 +1,10 @@
+use serde::{Deserialize, Serialize};
+
+pub type Version = String;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(tag = "status", rename_all = "lowercase")]
+pub enum Event {
+    Online { version: String },
```

Comment:
you forgot to your type alias. :)

Reply:
> you forgot to your type alias. :)

Not sure but looks like you forgot a verb :) Can you rephrase?

Reply:
> > you forgot to your type alias. :)
> 
> Not sure but looks like you forgot a verb :) Can you rephrase?

lol. "use" is the verb. You used `String`.

Reply:
Ah, yeah, I removed it because I used it to show that this String in an unnamed field is actually a version but now the field is named thanks to @jplatte 's suggestion.

Reply:
I think the typedef can then be removed, right?

Reply:
Yeah, it is removed.